index.html

```diff
@@ -16,23 +16,150 @@
+                        </div>
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 22.png" alt="open laptop and a mobile" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 2</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
+                            <div class="tech-item">Ruby on Rails</div>
+                            <div class="tech-item">Javascript</div>
+                        </div>
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 26.png" alt="open laptop tilted to right" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 3</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
+                            <div class="tech-item">Ruby on Rails</div>
+                            <div class="tech-item">Javascript</div>
+                        </div>
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 27.png" alt="open laptop front view" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 4</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
+                            <div class="tech-item">Ruby on Rails</div>
+                            <div class="tech-item">Javascript</div>
+                        </div>
+                        <button class="button-generic button-project">See this project</button>
+                    </div>
+                </div>
+                <!-- individual project tile -->
+                <div class="project-tile">
+                    <img src="res-images/Rectangle 46.png" alt="open laptop front view 2" class="project-thumbnail">
+                    <div class="project-details">
+                        <h3 class="project-title">Project 5</h3>
+                        <div class="tech-item-container">
+                            <div class="tech-item">HTML/CSS</div>
```

Comment:
[ ] The list of project technologies should be a list `ul` and each of them a `li`.kindly fix that 🤓